### ISSUE TITLE
Suggestion: Buff Fire Damage (the component)

### ISSUE THREAD

**Issue by ProfHugo** [Growlith1223/ArsMagica2#86]
Currently fire damage is outclassed by frost damage and even lightning damage by quite a hefty amount in the "true damage department". 

Fire Damage has a base damage of 6 and a base mana cost of 120 (20 mana/damage); Lightning Damage has a base damage of 12 and a base mana cost of 180 (15 mana/damage); Frost Damage has a base damage of 10 and a base mana cost of 80 (8 mana/damage). On top of the 150% higher efficiency frost damage have over fire, it also applies a slow to whatever it hits, which is just plain ridiculous. Not to mention nether mobs takes reduced damage from fire, but double damage from frost...

As you can see fire damage is completely overshadowed by other damage types, having ridiculously weak damage for such a hefty mana cost (also fire resistance armor chops the damage down very brutally, and fire resist potion COMPLETELY negates fire damage,). I think fire damage deserves a buff that makes it more comparable to other damage types.

One simple solution would be to buff the damage and/or reduce the mana cost, but that would eventually have fire damage labeled the "goto raw true dps" component and overall be unhealthy to game balance. One way to fix this is to differentiate it from other damage components, by adding utility to the spell that contains it. Here are some viable suggestions off the top of my head...

Have the damage and/or mana cost scales with burnout, which seems easy to implement and make sense thematically. Having it scale exponentially with burnout would give it a very unique identity of a component that "does a ton of damage, but after you casted a bunch of spells and if you still have some mana left" kind of thing, which is great. (Although this is kinda unbalanced since the burnout meter is hidden from opponents, so you wouldn't know if it would either tickles or instantly oneshots you)

One alternative would be to have it do amplified damage against enemies on fire, similar to magma slime crystal in TCon 2. This is also relatively easy to implement, and gives it an identity of "if you're on fire, this would hit very hard, run away!" kind of thing, which also makes sense thematically and it's more balanced since there are multiple ways to counterplay it (namely dip yourself in water or run the heck away).

In conclusion, fire damage is in a very weak spot right now, and it definitely needs a buff or rework that could make it more viable and fun, both for the caster and the target.

Thank you for your time and consideration.

**Comment by Zieg777** [comment]
My two cent rebuttal:

Fire damage is a spell that it is almost guaranteed every AM2 mage will make. It's required for making a black aurem. Every mage has it, so a lot of them use it. 

Lightning on the other hand is meant to do more damage, if the compendium is anything to go by. Lightning also has one very important drawback. Affinity. If you get over 40% affinity in lightning, don't touch water, or you lose all your mana. Also you ignite nearby TNT. Also you supercharge creepers. I avoid lightning purely out of not wanting the affinity.

Frost. Yeah, my goto mid-level attack spell is magic/frost damage (and various debuffs). Hits hard and almost everything will be weak to one or the other.

I'd immediately argue against a flat buff to power, simply because of the firestorm spell.

However. (rebuttal over)
I like the idea of the various elemental damages having their own special flavor though. We have ignite already, so I wouldn't say to have it ignite enemies. Extra damage to mobs on fire though would be good.

- Random suggestions for elemental damages:

1. - Force. Nothing. You magically threw a rock, thats all.
2. - Ice. Slow, Already there.
3. - Lightning. (ignore armor, or perhaps stronger damage against higher armor) Perhaps a half second of entangle? Been electrocuted before? Not fun.
4. - Arcane. (ignore armor) Mana drain. Not necessarily the mana stealing, but at least hurt the other mage.
5. - Fire. Higher damage on enemies on fire.

**Comment by ProfHugo** [comment]
Personally I think Arcane (magic) Damage is the most special of them all. One reason is due to how much of a variable it is in terms of damage depending on the condition.

- It does "true" damage, but magic resist chops down the damage heavily

- Two components apply this type of damage, magic damage and manablast (oh wait a sec the last fix to magic resist isn't applied to manablast YET)

- It does more damage if certain conditions are fulfilled, making initial setup crucial for maximum dps 
  - Tripled damage against charmed and summoned targets
  - Small damage amplification against target with high level of arcane affinity. 

Personally I think magic damage is a great example of a well balanced and fun damage type. It has plenty of headroom to play with, isn't overtly broken (hehe manablast is still broken in pvp without magic resist fix), and allows for so many combinations of spells to work with it. Other damage type, such a fire damage and lightning damage should also have these kinds of features that differentiates then from each other, so it's not just a situation where "frost damage has the highest dps, so my only damage spell is projectile frost damage" but something like "I'm a glass cannon mage, I have {projectile-charm} and {projectile-magic damage} to go with it. I also have other items that can strip armor so that I can do more damage" or "I'm a fire spell-sword mage. I have a magma-slime longsword modded with blaze powder along with {touch-fire damage} in my offhand in order to do high sustained damage" kind of situation. It's just way more fun having damage spells be different from each other.

**Comment by ProfHugo** [comment]
In fact, one of the worse offender in this is actually the component that's overlooked by many people, drown. It does the exact same amount of damage as lightning do (12 base damage) while costing as much mana as frost do. It literally does nothing but pure damage, similar to lightning, but cost a little less than half the mana...
